### PR TITLE
B3 Accounting Views

### PR DESCRIPTION
**Pull request by biyeun**
Migrates all accounting views to bootstrap 3

@nickpell for soft QA cc @millerdev 
screen shot 2016-02-25 at 1 35 51 pm
screen shot 2016-02-25 at 1 36 04 pm
screen shot 2016-02-25 at 1 36 13 pm
screen shot 2016-02-25 at 1 36 21 pm
screen shot 2016-02-25 at 1 36 31 pm
screen shot 2016-02-25 at 1 36 47 pm
screen shot 2016-02-25 at 1 37 10 pm
screen shot 2016-02-25 at 1 37 32 pm
screen shot 2016-02-25 at 1 37 50 pm
screen shot 2016-02-25 at 1 38 08 pm

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -1951,17 +1966,18 @@ def __init__(self, invoice, *args, **kwargs):
...
-                crispy.ButtonHolder(
+                hqcrispy.B3MultiField(
+                    'Balance Adjustments',
                     crispy.Button(
                         'submit',
                         'Adjust Balance',
                         data_toggle='modal',
                         data_target='#adjustBalanceModal-%d' % invoice.id,
-                        css_class=('disabled' if invoice.is_wire else '') + ' disable-on-submit',
+                        css_class=('btn-default disabled' if invoice.is_wire else 'btn-default') + ' disable-on-submit',
```
Comment (dimagimon) [inline corehq/apps/accounting/forms.py]: E501 line too long (120 > 115 characters)

**Comment by nickpell** [comment]
nice, is this on staging?

**Comment by millerdev** [comment]
cruizin

**Comment by nickpell** [comment]
@biyeun looks like the select2 widget for Client Contact Emails at https://staging.commcarehq.org/hq/accounting/accounts/7/ is no longer working

**Comment by biyeun** [comment]
@nickpell i'll put it up right now...

**Comment by nickpell** [comment]
@biyeun Also see select2 widget for "Transfer Subscription To" at https://staging.commcarehq.org/hq/accounting/subscriptions/1553/

**Comment by biyeun** [comment]
@nickpell updated

**Comment by nickpell** [comment]
> @biyeun Also see select2 widget for "Transfer Subscription To" at https://staging.commcarehq.org/hq/accounting/subscriptions/1553/

tested locally and on staging, still not autocompleting

**Comment by biyeun** [comment]
@nickpell updated (not on staging yet)

**Comment by nickpell** [comment]
@biyeun autocomplete for "Transfer Subscription To" is still broken (JS errors - see https://staging.commcarehq.org/hq/accounting/subscriptions/21/ for example)

**Comment by biyeun** [comment]
@nickpell did you try that locally? i wasn't able to deploy staging all day

**Comment by biyeun** [comment]
I added the fix right before I said that...

**Comment by nickpell** [comment]
working now, looks like the issue was the staging memory problems

**Pull request merged**